Conceptpower+Spring/src/test/java/edu/asu/conceptpower/core/ConceptManagerTest.java

```diff
@@ -113,13 +114,16 @@ public void testAddConceptListEntry() throws DictionaryDoesNotExistException, Di
 		newConcept.setPos("noun");
 		newConcept.setWord("test new");
 		newConcept.setWordnetId("WID-1");
-		
+
 		String id = managerToTest.addConceptListEntry(newConcept);
 		Assert.assertNotNull(id);
 	}
-	
+
+	@Rule
+	public ExpectedException expectedEx = ExpectedException.none();
```

Comment:
I think I'd move that declaration to the beginning of the class with all the other fields.